README.md

```diff
@@ -19,16 +27,30 @@ If you want to update the dependencies to a specific Polkadot SDK version, you c
 ```sh
 # Go to the directory containing the Cargo.toml file you want to update
 cd <cargo-toml-dir>
-# Run the psvm command to update the dependencies
+# Run the psvm command to update the dependencies using PSVM local versions
 psvm
 # You can also update it using the path to the Cargo.toml file
 psvm -p <cargo-toml-dir>/Cargo.toml
 # Overwrite local dependencies with crates.io versions
 psvm -o
-# Update to a specific Polkadot SDK version (default 1.3.0)
+# Update to a specific Polkadot SDK version (default 1.3.0) using PSVM local versions
 psvm -v "1.7.0"
+# Update to the versions defined in a specific Polkadot SDK branch Plan.toml file
+# i.e https://raw.githubusercontent.com/paritytech/polkadot-sdk/release-crates-io-v1.6.0/Plan.toml
+psvm -b "release-crates-io-v1.6.0"
+# Update to the versions defined in a specific Polkadot SDK branch Cargo.lock file
+# i.e https://raw.githubusercontent.com/paritytech/polkadot-sdk/release-crates-io-v1.3.0/Cargo.lock
+psvm -b "release-crates-io-v1.3.0" -s "Cargo.lock"
```

Comment:
It should probably understand the branch naming schema without help.

Reply:
Makes more sense, will require only version (i.e. "1.3.0") and always get from "release-crates-io-v" branches